### PR TITLE
Add office team

### PR DESCRIPTION
**Pull request by zoetrope**
Signed-off-by: zoetrope <a.ikezoe@gmail.com>

### CODE REVIEW

**Review by umezawatakeshi — APPROVED** [review]

**Comment by morimoto-cybozu** [comment]
@zoetrope 
I cannot see what you did.
Would you please tell me what commands you executed and which document you followed?

**Comment by zoetrope** [comment]
https://github.com/cybozu-private/neco-wiki/pull/678

File: team-management/template/settings.json [hunk 1]
```diff
@@ -321,6 +321,36 @@
...
+        ]
+      },
+      "delegates": [
+        {
+          "name": "maneki",
+          "roles": [
+            "admin"
+          ]
+        }
+      ],
```
Comment (morimoto-cybozu) [inline team-management/template/settings.json:337]: This block is not described in the procedure document.
Not all Neco members could supply this block just by reading the document and the request record.
The document should be enhanced.

File: team-management/template/settings.json [hunk 2]
```diff
@@ -321,6 +321,36 @@
         }
       ]
     },
+    "office": {
+      "argocd": {
+        "repositories": [
+          "https://github.com/cybozu-private/*"
```
Comment (morimoto-cybozu) [inline team-management/template/settings.json:327]: This URL pattern represents a wider range of repositories than requested.
Not all Neco members could put this pattern just by reading the document and the request record.
The document should be enhanced.

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: team-management/template/settings.json (hunk 1 above)
Comment (zoetrope) [reply team-management/template/settings.json:337]: added: https://github.com/cybozu-private/neco-wiki/pull/680

File: team-management/template/settings.json (hunk 2 above)
Comment (zoetrope) [reply team-management/template/settings.json:327]: added: https://github.com/cybozu-private/neco-wiki/pull/680

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**